app/controllers/obtain_annotations_with_callback_jobs_controller.rb

```diff
@@ -0,0 +1,74 @@
+class ObtainAnnotationsWithCallbackJobsController < ApplicationController
+  before_action :authenticate_root_user!, only: :new
+
+  def new
+    @project = Project.editable(current_user).find_by_name(params[:project_id])
+  end
+
+  def create
+    project = Project.editable(current_user).find_by_name(params[:project_id])
+    raise "Could not find the project: #{params[:project_id]}." unless project.present?
+    raise "Up to 10 jobs can be registered per a project. Please clean your jobs page." unless project.jobs.count < 10
+
+    # to determine the annotator
+    annotator = Annotator.find(params[:annotator])
+    raise "Could not find annotator: #{params[:annotator]}." unless annotator.present?
+
+    # to determine the options
+    options = {
+      mode: params[:mode],
+      prefix: annotator.name
+    }
+
+    # to determine the docids
+    options[:mode], docids, messages = docids_for(project, options[:mode])
```

Comment:
optionsを作るタイミングを入れ替えると綺麗に書けそうです。

```suggestion
    # to determine the docids
    mode, docids, messages = docids_for(project, params[:mode])

    # to determine the options
    options = {
      mode:,
      prefix: annotator.name
    }
```